features/scoped-custom-element-registries.yml

```
@@ -0,0 +1,11 @@
+name: Scoped custom element registries
+description: The `CustomElementRegistry()` constructor creates multiple custom element registries so that multiple custom element definitions for a single tag name can exist without conflicts. This is useful if more than one library is used, for example.
+spec: https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-api
```

Looking at the [HTML spec PR](https://github.com/whatwg/html/pull/10869), a [corresponding DOM spec PR](https://github.com/whatwg/dom/pull/1341) was mentioned. So I think we need to link to both specs here:

```suggestion
spec:
  - https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-api
  - https://dom.spec.whatwg.org/#element-custom-element-registry
```